src/pip/_vendor/tomli/__init__.py

```diff
@@ -0,0 +1,4 @@
+__all__ = ("loads", "load", "TOMLDecodeError")
+__version__ = "0.2.6"  # DO NOT EDIT THIS LINE MANUALLY. LET bump2version UTILITY DO IT
+
+from pip._vendor.tomli._parser import TOMLDecodeError, load, loads
```

Comment:
This import and the import on line 7 in ` src/pip/_vendor/tomli/_parser.py` are the only things I changed in the vendored `Tomli`.

Are these import changes significant enough that I need to document them in the vendoring README and add something to `tools/vendoring/patches` ?

Reply:
Yes. This needs to be called out in the README, and has to be explicitly patched in vendoring/patches as well.

Reply:
:+1: I added a note in the README. A patch in `vendoring/patches` seems to not be required? And looks to me like other vendored packages do similar changes without having a patch...

Tell me if I'm doing something wrong :smile:

Reply:
Ok I think I figured this out by myself. No patch is needed because no change was made compared to the what the automation creates.

In the first commit I had no clue how this fancy automation works, so dumb me added the package manually, and made precisely the same changes that the automation would've done.

Reply:
Reverted the README addition, as I now understand I did not make changes to what the automation produces.